packages/authenticator/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -69,6 +71,9 @@ export 'src/widgets/form.dart'
         ConfirmSignInCustomAuthForm,
         ConfirmSignInMFAForm,
         ConfirmSignInNewPasswordForm,
+        ContinueSignInWithMfaSelectionForm,
+        ContinueSignInWithTotpSetupForm,
+        ConfirmSignInWithTotpMfaCodeForm,
```

Comment:
Don't think these need to be exported

Reply:
This is following our export patterns to allow customer customizations. 

Use case: Customer wants to add their logo to the new TOTP screens.

Further granularity is offered as we export the individual form fields themselves under the class `ConfirmSignInFormField`. 

Use case: Customer wants to rearrange the order form fields appear in the UI.

Reply:
I don't see much value exporting these given that they can't be customized.